common/src/admin/js/update-notice.js

```diff
@@ -0,0 +1,11 @@
+jQuery(function ($) {
+    $(document).on('click', `.${notice_params.class} .notice-dismiss`, function () {
```

Comment:
You only need to wrap code in `jQuery(function($) { ... })` if you need to wait for the DOM to load.  In this case it's not needed because you are attaching the event handler on `document`, which is always loaded.